lib/config/config-initializer.js

```diff
@@ -486,18 +489,35 @@ function promptUser() {
         // continue with the questions otherwise...
         return inquirer.prompt([
             {
-                type: "confirm",
-                name: "es6",
-                message: "Are you using ECMAScript 6 features?",
-                default: false
+                type: "list",
+                name: "ecmaVersion",
+                message: "Which version of ECMAScript do you use?",
+                choices: [
+                    { name: "ES3", value: 3 },
+                    { name: "ES5", value: 5 },
+                    { name: "ES2015", value: 2015 },
+                    { name: "ES2016", value: 2016 },
+                    { name: "ES2017", value: 2017 },
+                    { name: "ES2018", value: 2018 }
+                ],
+                default: 1 // This is the index in the choices list
             },
             {
                 type: "confirm",
                 name: "modules",
                 message: "Are you using ES6 modules?",
                 default: false,
                 when(answers) {
-                    return answers.es6 === true;
+                    return answers.ecmaVersion >= 2015;
+                }
+            },
+            {
+                type: "confirm",
+                name: "es6Globals",
+                message: "Are you using ES6 globals (such as `Map` and `Set`)?",
```

Comment:
Can we reword this as "Do you want to enable..." rather than "Are you using..."? It seems like someone might answer "no" because they're currently not using `Map` and `Set`, even though they might want to do so in the future.

Reply:
I'll will update it. BTW, this sentence is copied, not written by me.